org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerTestPlugin.java

```diff
@@ -38,8 +43,20 @@ public class JavaLanguageServerTestPlugin implements BundleActivator {
 	 */
 	@Override
 	public void start(BundleContext context) throws Exception {
-		IEclipsePreferences prefs = InstanceScope.INSTANCE.getNode(MavenJdtPlugin.PLUGIN_ID);
-		prefs.put(PREFERENCE_LOOKUP_JVM_IN_TOOLCHAINS, Boolean.FALSE.toString());
+		// https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/3238
+		if ("false".equals(System.getProperty(PREFERENCE_LOOKUP_JVM_IN_TOOLCHAINS, "true"))) {
+			IEclipsePreferences prefs = InstanceScope.INSTANCE.getNode(MavenJdtPlugin.PLUGIN_ID);
+			prefs.put(PREFERENCE_LOOKUP_JVM_IN_TOOLCHAINS, Boolean.FALSE.toString());
+		}
+		try {
+			long start = System.currentTimeMillis();
+			String symbolicName = MavenJdtPlugin.PLUGIN_ID;
+			JavaLanguageServerPlugin.debugTrace("Starting " + symbolicName);
+			Platform.getBundle(symbolicName).start(Bundle.START_TRANSIENT);
+			JavaLanguageServerPlugin.logInfo("Started " + symbolicName + " " + (System.currentTimeMillis() - start) + "ms");
```

Comment:
Yes, it is. we need to ensure that the JVMs are created. See https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/3240/commits/af553257c8efeb65238755b5ad1b2b3009c46b2f#diff-f4fb57a3a0f54cb8ba5055f167792f7e64118f26aa18c7b1ecafea10225c6e94R144

Reply:
As a note, I think the reason `MavenJdtPlugin.PLUGIN_ID;` fails to trigger bundle activator of `o.e.m2e.jdt` bundle  is because `PLUGIN_ID` is being inlined. Calling `MavenJdtPlugin.getDefault()` definitely works, but better to be explicit.